.github/workflows/python.yml

```diff
@@ -115,6 +115,7 @@ jobs:
       - uses: actions/setup-python@v5
         with:
           python-version: "3.9"
+      - run: rustup component add rustfmt
```

Comment:
should we use `setup-rust` instead? this feels a bit out of place

Reply:
Oh yeah, much better